src/OrchardCore.Modules/OrchardCore.Media/OrchardHelperExtensions.cs

```diff
@@ -0,0 +1,23 @@
+using System.Collections.Generic;
+
+namespace OrchardCore.Media
```

Comment:
normally no name space on the orchard helper methods

Reply:
@agriffard please remove the namespace as Dean said